upgrade/upgrade-scripts/common.sh

```diff
@@ -204,3 +204,10 @@ function set_upgrade_property() {
 	source_upgrade_properties ||
 		die "failed to read properties file after setting '$1=$2'"
 }
+
+function turn_verbose_logging_on() {
+    exec 4> >(logger -t $0)
+    BASH_XTRACEFD="4"
+    PS4=':${BASH_SOURCE}:${LINENO}+'
```

Comment:
How would you feel about changing this to:
```
    exec 4> >(logger -t "delphix-upgrade[$BASHPID]")
    BASH_XTRACEFD="4"
    PS4='${BASH_SOURCE}:${LINENO} '
```
?

This would result in message like the following:
```
Nov 20 23:06:37 ps-trunk delphix-upgrade[5772]: /var/dlpx-update/latest/upgrade:103 case "$UPGRADE_TYPE" in
Nov 20 23:06:37 ps-trunk delphix-upgrade[5772]: /var/dlpx-update/latest/upgrade:111 CONTAINER=delphix.mPIq2s4
Nov 20 23:06:37 ps-trunk delphix-upgrade[5772]: /var/dlpx-update/latest/upgrade:112 [[ -n delphix.mPIq2s4 ]]
Nov 20 23:06:37 ps-trunk delphix-upgrade[5772]: /var/dlpx-update/latest/upgrade:114 /var/dlpx-update/6.1.0.0-snapshot.20191114231533366+jenkins-selfservice-devops-gate-master-appliance-build-master-pre-push-3703/upgrade-container start delphix.mPIq2s4
```

If we don't include the PID in the tag used with `logger`, I think it'll be difficult to discern to different processes both attempting to upgrade concurrently. Obviously this shouldn't occur, but it's not impossible (e.g. when running upgrade via the scripts), so it'd be nice to account for that.